sdk/cosmos/azure-cosmos-tests/src/test/java/com/azure/cosmos/CosmosItemTest.java

```diff
@@ -317,6 +317,36 @@ public void readManyWithSingleTuple() throws Exception {
         }
     }
 
+    @Test(groups = { "simple" }, timeOut = TIMEOUT)
+    public void readManyWithNonExistentIds() throws Exception {
```

Comment:
This test only simulates the read-via-point-read case. Can you add one please where we would effectively read via query as well - to ensure the behavior is identical?

Reply:
Yeah - rectified it in next iterations.